lib/scorecard.rb

```diff
@@ -0,0 +1,32 @@
+class Scorecard
+  attr_reader :frames
+
+  def initialize
+    @frames = []
+  end
+
+  def update(frame, frames)
+    frame.total = frame.rolls.sum(&:score)
+    if frame.rolls.size == 2 || frame.rolls.map(&:score).sum >= 10
+      @frames << frame
+    end
+  end
```

Comment:
see line 25 on game.rb, which is very similar. I think scorecard and game classes are slightly creeping into each other's territory, resulting in duplicating information / code

Reply:
Agreed.  I will take the time to refactor the classes and ensure the functions are seperate.